charts/zeebe-benchmark/values.yaml

```diff
@@ -336,6 +336,25 @@ camunda-platform:
 
   operate:
     enabled: false
+    image:
+      tag: 8.2.8
```

Comment:
:thought_balloon: Do we really want to fix the Operate version? Shouldn't we use the same version as we use for the rest of camunda-platform?

Reply:
Actually we don't want to fix it, but we might need to set it separately in the benchmark workflows. We also need to make sure they can create docker images on the fly. Which is an open TODO https://github.com/camunda/operate/issues/5178